examples/electron-todo-list/src/renderer/atlas-app-services/config.json

```diff
@@ -0,0 +1,3 @@
+{
+  "ATLAS_APP_ID": "example-wasm-nwvek"
```

Comment:
Should we replace with `YOUR-APP-ID-HERE`?

Reply:
I agree 👍 (To grab from the README instructions, we could use the placeholder `"YOUR_APP_ID"`.)

Reply:
For sure, I did not intend to commit an actual app-id here